src/tests/compiler/expression/object/propertyAssignmentTests.ts

```diff
@@ -30,4 +31,40 @@ describe(nameof(PropertyAssignment), () => {
             expect(sourceFile.getFullText()).to.equal("const t = { prop: { t } as string, prop2: 6 };");
         });
     });
+
+    describe(nameof<ObjectLiteralElementLike>(p => p.remove), () => {
+        function doTest(code: string, propertyToRemove: string | ((p: Node) => boolean), expectedCode: string) {
+            const {sourceFile, descendant} = getInfoFromTextWithDescendant<ObjectLiteralExpression>(code,
+                SyntaxKind.ObjectLiteralExpression);
+            if(typeof propertyToRemove === "string")
+                descendant.getPropertyOrThrow(propertyToRemove).remove();
+            else
+                descendant.getProperties().find(propertyToRemove)!.remove();
+            expect(descendant.getText()).to.equal(expectedCode);
+        }
+
+        it("should remove first property assignment", () => {
+            doTest("const t = { prop1: 5, prop2: 6 };", "prop1", "{ prop2: 6 }");
+        });
+
+        it("should remove last property assignment and preserve trailing comma", () => {
+            doTest("const t = { prop1: [1,2,3], prop2: [['hello']], };", "prop2", "{ prop1: [1,2,3], }");
+        });
+
+        it("should remove property assignment in the middle", () => {
+            doTest("const t = { prop1: {prop1: 2}, prop2: {prop2: 3}, foo: {bar: '98989'} };", "prop2",
+                "{ prop1: {prop1: 2}, foo: {bar: '98989'} }");
+        });
+
+        it("should remove ShorthandPropertyAssignment", () => {
```

Comment:
This file is for the tests for `PropertyAssignment`. Could you move these tests into the appropriate file?